GOVERNANCE.md

```diff
@@ -69,8 +69,8 @@ Members participate in a vote.
 
 As per the [CPC charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md), the non-impact projects elect 2 voting CPC members:
 
-> up to two (2) Voting members may be nominated by the non-impact projects
-  based on a process set by the CPC.
+> up to two (2) Voting members may be nominated by the non-impact projects based on a process set by the CPC.
```

Comment:
We may need to wait for https://github.com/openjs-foundation/cross-project-council/pull/302 to land if we don't want to wait 14 days (due to modifying GOVERNANCE.md

Reply:
hmm.. makes sense 👍 Hopefully it will be merged.